plugins/patriot-aws/lib/patriot_aws/command/s3.rb

```diff
@@ -0,0 +1,248 @@
+module PatriotAWS
+  module Command
+    class S3Command < Patriot::Command::Base
+      declare_command_name :s3
+      include PatriotAWS::Ext::AWS
+
+      command_attr :name, :name_suffix, :inifile,
+                   :command, :src, :dest, :options
+
+      MODE_UPLOAD_FROM_LOCAL_TO_S3 = :mode_upload_from_local_to_s3
+      S3_PROTOCOLS = %w(s3 s3n s3a).freeze
+      COMMAND_COPY = :copy
+      S3_COMMANDS = [COMMAND_COPY].freeze
+
+      def job_id
+        job_id = "#{command_name}_#{@command}_#{@name}"
+        job_id += "_#{@name_suffix}" if @name_suffix
+        job_id
+      end
+
+      # @see Patriot::Command::Base#configure
+      def configure
```

Comment:
This is not needed if do nothing.
It's worth to set default name_suffix from `@target_datetime`.